torch/utils/tensorboard/summary.py

```diff
@@ -18,6 +18,8 @@
 from tensorboard.plugins.text.plugin_data_pb2 import TextPluginData
 from tensorboard.plugins.pr_curve.plugin_data_pb2 import PrCurvePluginData
 from tensorboard.plugins.custom_scalar import layout_pb2
+from tensorboard.plugins.mesh.plugin_data_pb2 import MeshPluginData
```

Comment:
Looks like CI fails with

```
May 18 17:29:47 Traceback (most recent call last):
May 18 17:29:47   File "test_tensorboard.py", line 56, in <module>
May 18 17:29:47     from torch.utils.tensorboard import summary, SummaryWriter
May 18 17:29:47   File "/opt/python/2.7/lib/python2.7/site-packages/torch/utils/tensorboard/__init__.py", line 6, in <module>
May 18 17:29:47     from .writer import FileWriter, SummaryWriter  # noqa F401
May 18 17:29:47   File "/opt/python/2.7/lib/python2.7/site-packages/torch/utils/tensorboard/writer.py", line 22, in <module>
May 18 17:29:47     from .summary import (
May 18 17:29:47   File "/opt/python/2.7/lib/python2.7/site-packages/torch/utils/tensorboard/summary.py", line 21, in <module>
May 18 17:29:47     from tensorboard.plugins.mesh.plugin_data_pb2 import MeshPluginData
May 18 17:29:47 ImportError: No module named mesh.plugin_data_pb2
May 18 17:29:47 Traceback (most recent call last):
May 18 17:29:47   File "test/run_test.py", line 440, in <module>
May 18 17:29:47     main()
May 18 17:29:47   File "test/run_test.py", line 432, in main
May 18 17:29:47     raise RuntimeError(message)
May 18 17:29:47 RuntimeError: test_tensorboard failed!
```

@lanpa would you be interested in updating our pinned package for CI tests in this PR?

Reply:
Also, I think we should move this and the next import into the `def mesh(...):` function below. We should do that for all proto imports, since they should be isolated in use.

Reply:
CI fails because this feature is only in tb-nightly at this time (will be in tf 1.14). Do we have to wait for it?

> Also, I think we should move this and the next import into the `def mesh(...):` function below. We should do that for all proto imports, since they should be isolated in use.

I'm not sure if it would have a significant performance impact.